rd-net/Lifetimes/Lifetimes/LifetimeDefinition.cs

```diff
@@ -160,34 +189,57 @@ public bool AllowTerminationUnderExecution
       [PublicAPI] set => ourAllowTerminationUnderExecutionSlice.InterlockedUpdate(ref myState, value);
     }
     
+    public LifetimeTerminationTimeout TerminationTimeout => ourTerminationTimeoutSlice[myState];
+    
     #endregion
 
     
     
     #region Init
-    
+
     /// <summary>
-    /// Creates toplevel lifetime definition with no parent. <see cref="Status"/> will always be <see cref="LifetimeStatus.Alive"/>.
+    /// Creates toplevel lifetime definition with no parent. <see cref="Status"/> will always be <see cref="LifetimeStatus.Alive"/>.<br/>
+    /// Created definition and all its children (if not explicitly overriden) will have specified termination timeout (see <see cref="LifetimeTerminationTimeout"/>). 
     /// </summary>
```

Comment:
Why put termination timeout as contructor parameter. It's a kind of rare thing that make API less understandable.

You can make property for LifetimeDefinition:

LifetimeTerminationTimeoutKind TerminationTimeout { get; set;}

Reply:
Because termination timeout inherited by nested lifetimes (if not explicitly overridden). The writable property approach much harder to implement in current design and it harder to understand in edge cases, e.g. what should happen if termination timeout set to parent lifetime while nested one already in terminating state?

Reply:
When Termination Timeout is set via property, it shouldn't autimatically propagate to already existing children. Only new children can inherit Timeout.
You can imagine the most typical pattern `new LifetimeDefinition() { TerminationTimeout = Long}`

Reply:
Writable property contract doesn't suppose exactly such behavior and only way to declare it is xmldoc (but who reads the docs?) On other hand, read-only property doesn't have this behavior ambiguity. However I respect your opinion as library author and will re-work to writable property if you insist.

Reply:
done